src/install.c

```diff
@@ -550,6 +550,43 @@ static gboolean verify_compatible(RaucInstallArgs *args, RaucManifest *manifest,
 	}
 }
 
+static gboolean check_version_limits(RaucInstallArgs *args, RaucManifest *manifest, GError **error)
+{
+	GError *ierror = NULL;
+
+	if (args->ignore_version_limit)
```

Comment:
Should this get a `g_message()` printout about explicitly disabling version check?

Maybe ordered then after the test for `system_min_bundle_version`.

Reply:
While we don't have that for the compatible check, it's probably a good idea.